plugins/together/newhelm/suts/together_client.py

```diff
@@ -42,7 +42,15 @@ def _retrying_post(url, headers, json_payload):
     retries = Retry(
         total=6,
         backoff_factor=2,
-        status_forcelist=[429, 503, 524],
+        status_forcelist=[
+            408,  # Request Timeout
+            421,  # Misdirected Request
+            423,  # Locked
+            424,  # Failed Dependency
+            425,  # Too Early
+            429,  # Too Many Requests
+        ]
+        + list(range(500, 599)),  # Add all 5XX.
```

Comment:
> I couldn't find a cleaner way to tell urllib to do this.

Not sure if this might be helpful but `requests` provides a list of status codes
https://github.com/psf/requests/blob/8dd3b26bf59808de24fd654699f592abf6de581e/src/requests/status_codes.py

What you already have is swell as well